### PR TITLE
fixes the bluespace scanner oddity

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
Makes the bluespace scanner device actually work. Removes the check, because I couldn't make it work and frankly i dunno if it really is necessary.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by king5327** [comment]
Looking at it, I genuinely don't see why the check needs to be there. Like, what purpose does locking it to the ultimate mega-geniuses of the colony serve?

**Comment by cdb-is-not-good** [comment]
> Looking at it, I genuinely don't see why the check needs to be there. Like, what purpose does locking it to the ultimate mega-geniuses of the colony serve?

I Don't hate the idea of it. Having a check for flavor reasons. But it doesn't work and I don't really feel like fucking with it, It doesn't really matter more than for aesthetic reasons

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -666,19 +666,13 @@
...
-	if(!iscarbon(user) || !issilicon(user))
-		return//Prevents ghosts form making a runtime
-	if(!user.stats?.getPerk(PERK_SI_SCI) || !usr.stat_check(STAT_COG, 60)) //got to be smarts
-		to_chat(usr, SPAN_WARNING("This tool is far too complex for you to comprehend how to even use it. The data and formulas displayed look like complete alien gibberish."))
-		return
+/obj/item/oddity/si_bluespace_scanner/attack_self(mob/user as mob)
 	var/area/my_area = get_area(src)
 	if(my_area.bluespace_entropy)
-		to_chat(user, SPAN_NOTICE("The Tuning Device measures bluespace entropy in this room to be [my_area.bluespace_entropy] zeframs.")) // Considering Bluespace Entropy the same as Subspace Distortion, and not making the reference more obvious than it is.
+		to_chat(user, SPAN_NOTICE("The Nullifier's scanner is detecting bluespace entropy in this room to be [my_area.bluespace_entropy] zeframs."))
```
Comment (ThePainkiller) [inline code/game/objects/items/oddities.dm:672]: This is not a nullifier, which is the structure, this is the Bluespace Tuning Device....

Comment (cdb-is-not-good) [reply code/game/objects/items/oddities.dm:672]: loloops

**Pull request merged**